data/strings/strings.txt

```diff
@@ -7370,86 +7370,93 @@
+    vi = Dịch vụ vị trí của Google Play
+    zh-Hans = Google Play 定位服务
+    zh-Hant = Google Play 定位服務
 
   [pref_use_google_play]
     comment = Preference text
     tags = android
-    en = Use Google Play Services to determine your current location
-    af = Gebruik Google Play-dienste om u huidige ligging te bepaal
-    ar = استخدام خدمات Google Play لتحديد موقعك الحالي
-    be = Выкарыстоўваць сэрвісы Google Play для вызначэння месцазнаходжання
-    bg = Използвайте услугите на Google Play, за да определите текущото си местоположение
-    ca = Usa els serveis de Google Play per a obtenir la ubicació actual
-    cs = Používat Služby Google Play pro získání aktuální polohy
-    da = Anvend Google Play Services til at få din aktuelle placering
-    de = Nutzen Sie die Google Play-Dienste, um Ihren aktuellen Standort zu erhalten
-    el = Χρησιμοποιήστε υπηρεσίες Google Play για να προσδιορίσετε την τρέχουσα τοποθεσία σας
-    es = Usar los servicios de Google Play para obtener su ubicación actual
-    et = Kasuta Google Play teenuseid oma hetkeasukoha määramiseks
-    eu = Erabili Google Play zerbitzuak zure uneko kokapena zehazteko
-    fa = استفاده از Google Play Services برای تعیین موقعیت کنونی شما
-    fi = Käytä Google Play -palveluita sijaintisi selvittämiseksi
-    fr = Utilisez les Services Google Play pour obtenir votre emplacement actuel
-    hu = Használja a Google Play szolgáltatást, hogy szert tegyen aktuális helyszínéről
-    id = Gunakan Layanan Google Play untuk mendapatkan lokasi Anda saat ini
-    it = Usa i servizi Google Play per ottenere la tua posizione attuale
-    ja = 現在のロケーションを取得するためにGoogle Playのサービスを使う
-    ko = Google Play 서비스를 사용하여 현재 위치 정보 얻기
-    lt = Naudoti Google Play paslaugas dabartinės vietos nustatymui
-    mr = तुमचे वर्तमान स्थान निर्धारित करण्यासाठी Google Play सेवा वापरा
-    nb = Bruk Google Play Services for å hente din nåværende posisjon
-    nl = Gebruik de diensten van Google Play om uw huidige locatie te bepalen
-    pl = Używa usług Google Play do ustalenia aktualnego położenia
-    pt = Use os Serviços Google Play para determinar a sua localização atual
-    pt-BR = Usar Serviços do Google Play para determinar a sua localização atual
-    ro = Folosește serviciile Google Play pentru a obține poziția actuală
-    ru = Использовать Google Play Services для определения позиции
-    sk = Pomocou Google Play služieb získajte svoju aktuálnu polohu
-    sv = Använd Google Play Services för att bestämma din aktuella position
-    th = ใช้บริการ Google Play เพื่อรับตำแหน่งปัจจุบันของคุณ
-    tr = Mevcut konumunuzu belirlemek için Google Play Hizmetlerini kullanın
-    uk = Щоб визначити ваше поточне місцезнаходження, використовуйте сервіси Google Play
-    vi = Sử dụng Dịch vụ Google Play để có được vị trí hiện tại của bạn
-    zh-Hans = 使用Google Play服务以获取您的当前位置。
-    zh-Hant = 使用 Google Play 服務來獲得您的目前位置
+    en = Quickly determine your approximate location using Bluetooth, WiFi, or mobile network
```

Comment:
This is somewhat misleading as the network location could be used regardless of the GPlay services enabled or not.

Reply:
In reality, without google services, it doesn't work.

On fdroid version this setting is not displayed. 

@rtsisyk is it possible to detect the presence of GP Services and show/hide this setting?